projects/js-packages/components/components/boost-score-graph/annotations-plugin.ts

```diff
@@ -0,0 +1,70 @@
+import uPlot from 'uplot';
+import { Annotation } from '.';
+
+import './style-annotation.scss';
```

Comment:
I was thinking whether it's a good idea to use global styles or local, but this way allows easily adding custom styles and overriding the existing ones.